src/test/java/com/epam/reportportal/listeners/ListenerUtilsTest.java

```diff
@@ -34,4 +40,23 @@ public void getLaunchMode() {
 		Assert.assertEquals(DEFAULT, ListenersUtils.getLaunchMode("notvalid"));
 		Assert.assertEquals(DEBUG, ListenersUtils.getLaunchMode("Debug"));
 	}
+
+	@Test
+	public void handleException_restEndpointIOExceptionAndNullLogger() {
```

Comment:
Basically this test checks the case when the "logger" method argument is null just to be sure that the error messaging code is reachable:
```
if (logger != null) {
       logger.error(message, exception);
} else {
       System.out.println(exception.getMessage());
}
```
And I agree that it makes sense to cover the code branch with output to logger too.